getting-started/installation-timescale-forge.md

```diff
@@ -0,0 +1,26 @@
+>:WARNING: Our scaling out capabilities are currently in BETA and
```

Comment:
I think we should remove this warning on multi-node?  Not sure why it's specific to Forge?